public/angular/translations/locale-en.json

```diff
@@ -19,6 +19,8 @@
   "rss": "RSS",
   "elmo": "Elmo",
   "facebook": "Facebook",
+  "smsgh": "SMSGh",
+  "Smsgh": "Smsgh",
```

Comment:
I would like to do that too, but I have no idea if something else uses "smsgh". I added line 23 because the warning said that "Smsgh" translation not found.